system-x/services/db/common/src/main/java/software/tnb/db/common/openshift/OpenshiftDB.java

```diff
@@ -96,6 +112,8 @@ public void create() {
 
     @Override
     public void undeploy() {
+        SecurityContextConstraints scc = OpenshiftClient.get().securityContextConstraints().withName(SCC_NAME).edit();
+        scc.getGroups().remove("system:serviceaccounts:" + OpenshiftConfiguration.openshiftNamespace());
```

Comment:
you also need to patch it the same way as in create

Reply:
You're right - changed